docs/Downloaders/qBittorrent/Tips/How-to-run-the-unRaid-mover-for-qBittorrent.md

```diff
@@ -137,33 +137,8 @@ You can choose one of the following 3 options (select a tab) to install `qbittor
 
 ### Script
 
-Now, using your favorite text editor ([VSCode](https://code.visualstudio.com/){:target="_blank" rel="noopener noreferrer"}/[Notepad++](https://notepad-plus-plus.org/downloads/){:target="_blank" rel="noopener noreferrer"}) edit the script you downloaded at the beginning of the guide ([HERE](#qbit-mover-script)).
-
-You only need to change a few options at the top of the script.
-
-```python
-# --DEFINE VARIABLES--#
-# Set Number of Days to stop torrents between two offsets
-# days_from set to 0 will pause any torrents from todays date
-# days_to will be the upper limit of how far you want to pause torrents to
-days_from = 0
-days_to = 2
-qbt_host = "192.168.2.200:8080"
-qbt_user = None
-qbt_pass = None
-# --DEFINE VARIABLES--#
-```
-
-- `days_from` => Set Number of Days to stop torrents **from** for the move.
-- `days_to` => Set Number of Days to stop torrents **to** for the move.
-- `qbt_host` => The URL you use to access qBittorrent locally. (*the* `"` *should remain*)
-- `qbt_user` => Your used qBittorrent `User Name` if you have authentication enabled. Add `'` either side.
-- `qbt_pass` => Your used qBittorrent `Password` if you have authentication enabled. Add `'` either side.
```

Comment:
It might also be a good idea to add some form of description for the arguments used now and changing them would be necessary to use this...

Reply:
there is also native description per script argument 😉

Reply:
Yea i dismissed it, I didn't see that you moved it to the bottom, not sure why you moved it there but as long as it is there.